gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1042,9 +1043,15 @@ void gnc_file_ofx_import (GtkWindow *parent)
     DEBUG("gnc_file_ofx_import(): Begin...\n");
 
     default_dir = gnc_get_default_directory(GNC_PREFS_GROUP);
+    GList *filters = NULL;
+    GtkFileFilter* filter = gtk_file_filter_new ();
+    gtk_file_filter_set_name (filter, "ofx/qfx files (*.ofx,*.qfx)");
+    gtk_file_filter_add_pattern (filter, "*.[oqOQ][fF][xX]");
+    filters = g_list_append( filters, filter );
```

Comment:
It doesn't matter here; but as a matter of habit we'd want to use `g_list_prepend`. See https://developer.gnome.org/glib/stable/glib-Doubly-Linked-Lists.html#g-list-append

Reply:
OK I'll change it for good practice